src/core/llm.py

```diff
@@ -20,11 +20,14 @@
     GroqModelName,
     OllamaModelName,
     OpenAIModelName,
+    AzureOpenAIModelName,
```

Comment:
Need to run linter / formatter ideally through the pre-commit, or CI will keep failing. See instructions in README under Contributing